ExperimentTwitter/User+Twitter.h

```diff
@@ -0,0 +1,19 @@
+//
+//  User+Twitter.h
+//  ExperimentTwitter
+//
+//  Created by ayur.j on 11/09/17.
+//  Copyright © 2017 ayur.j. All rights reserved.
+//
+
+#import "User+CoreDataClass.h"
+
+extern NSString * const UserDidLogInNotification;
+extern NSString * const UserDidLogOutNotification;
```

Comment:
No space after asterix

Reply:
I think space should be present between "*" and "const". The pods I've used also uses  space between "*" and "const"

Reply:
Yes that's debatable but in PingPong, you would not find space after asterix mostly. So it's a 'conventions' thing.